packages/web/src/application/test/zoom/zoom-provider.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactNode } from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { User } from '@chess-tent/models';
+import application from '@application';
+import { users, zoom } from '../fixtures';
+
+const { components } = application;
+
+interface RenderOptions {
+  children: ReactNode;
+  user: User;
+  meetingNumber?: string;
+  queryCode?: boolean;
+}
+
+const renderInsideProvider = ({
```

Comment:
Rename it to `renderWithProvider`

Also change signature to `renderWithProvider(children, options)`

This way you're going to use it `renderWithProvider(<Zoom/>, {})`.